schemas/appeal.schema.json

```diff
@@ -0,0 +1,555 @@
+    "caseValidationDate": {
+      "type": ["string", "null"],
+      "format": "date-time",
+      "examples": ["2023-07-27T20:30:00.000Z"],
+      "description": "The date the appeal was validated in the back-office"
+    },
+    "caseValidationOutcome": {
+      "type": ["string", "null"],
+      "enum": ["valid", "invalid", "incomplete"],
+      "description": "The outcome of the validation action"
+    },
+    "caseValidationInvalidDetails": {
+      "type": ["array"],
+      "items": {
+        "type": "string",
+        "examples": [
+          "Appeal has not been submitted on time",
+          "The appellant doesn't have the right to appeal"
+        ]
+      },
+      "description": "A list of reasons why the appeal is invalid",
+      "$comment": "The list will contain items only if the caseValidationOutcome is invalid"
+    },
+    "caseValidationIncompleteDetails": {
+      "type": ["array"],
+      "items": {
+        "type": "string",
+        "examples": [
+          "Appellant name is not the same on the application form and appeal form",
+          "LPA's decision notice is missing"
+        ]
+      },
+      "description": "A list of reasons why the appeal is incomplete",
+      "$comment": "The list will contain items only if the caseValidationOutcome is incomplete"
+    },
+    "caseExtensionDate": {
+      "type": ["string", "null"],
+      "format": "date-time",
+      "examples": ["2023-07-27T20:30:00.000Z"],
+      "description": "When the validation outcome is incomplete, an extension may be granted to provide missing information"
+    },
+    "caseStartedDate": {
+      "type": ["string", "null"],
+      "format": "date-time",
+      "examples": ["2023-07-27T20:30:00.000Z"],
+      "description": "A date indicating when the case was started, resulting in the creation of a timetable"
+    },
+    "casePublishedDate": {
+      "type": ["string", "null"],
+      "format": "date-time",
+      "examples": ["2023-07-27T20:30:00.000Z"],
+      "description": "A date indicating when the case was published"
+    },
+    "linkedCaseStatus": {
+      "type": ["string", "null"],
+      "enum": ["lead", "child", null],
+      "description": "Indicates if the case is linked, and the type of relationship"
+    },
+    "leadCaseReference": {
+      "type": ["string", "null"],
+      "examples": ["7002471"],
+      "description": "The reference of the lead case, if the case is a child case"
+    },
+    "lpaQuestionnaireDueDate": {
+      "type": ["string", "null"],
+      "format": "date-time",
+      "examples": ["2023-07-27T20:30:00.000Z"],
+      "description": "If the case is started and has a timetable, a deadline for the LPA to provide a response"
+    },
+    "lpaQuestionnaireSubmittedDate": {
+      "type": ["string", "null"],
+      "format": "date-time",
+      "examples": ["2023-07-27T20:30:00.000Z"],
+      "description": "The date the LPA provided a response to the case"
+    },
+    "lpaQuestionnaireCreateDate": {
```

Comment:
This should be `lpaQuestionnaireCreatedDate` (Created, which matches the other past-tense fields)